modules/nf-core/kmcp/index/meta.yml

```
@@ -0,0 +1,47 @@
+name: "kmcp_index"
+description: write your description here
```

Missing description of subcommand
```suggestion
description: Construct KMCP database from k-mer files
```